Essentials/Commands/WorldModule.cs

```diff
@@ -96,6 +96,70 @@ public void CleanFactions(int memberCount = 1)
             Context.Respond($"Removed {count} factions with fewer than {memberCount} members.");
         }
 
+        [Command("faction list", "lists all factions in the game")]
+        [Permission(MyPromoteLevel.Admin)]
+        public void ListFactions()
+        {
+            StringBuilder sb = new StringBuilder();
+            foreach(var faction in MySession.Static.Factions.ToList())
+            {
+                sb.Append($"{faction.Value.Tag}({faction.Value.Members.Count}) ");
+            }
+            Context.Respond(sb.ToString());
```

Comment:
This should definitely be a dialog. If the response is too long, it just overflows the chat box and is useless.
You can check in the Context object and see if the command was sent from the Torch UI. In that case, use Context.Respond. Otherwise send a DialogMessage